shared/chat/audio/audio-starter.native.tsx

```diff
@@ -131,6 +132,7 @@ const AudioStarter = (props: AudioStarterProps) => {
         simultaneousHandlers={panRef}
       >
         <Kb.PanGestureHandler
+          shouldCancelWhenOutside={false}
```

Comment:
What does this help with?

Reply:
https://software-mansion.github.io/react-native-gesture-handler/docs/handler-common.html#shouldcancelwhenoutside